salt/states/x509.py

```diff
@@ -644,6 +638,9 @@ def certificate_managed(
             - backup: True
 
     """
+    if "managed_private_key" in kwargs:
```

Comment:
Why are we adding a message here? This kwarg has already gone down the deprecation path and should be removed in this release.